packages/foreman/foreman/foreman.provreq

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python2
```

Comment:
Should this be `/usr/bin/env python` or some equivalent to future proof it?

Reply:
I am not sure. The existing scripts I see use `/usr/bin/python` and `/usr/bin/python3`.

Reply:
The code currently uses python-2 style code. We can make it python 3 compatible as well as python 2. Probably a good idea.